sigs/fedora/cluster.md

```diff
@@ -0,0 +1,50 @@
+# Fedora Konflux Cluster
+
+This gist is a list of notes about the Fedora Konflux cluster.
```

Comment:
This is technically no longer a gist. :)

Reply:
Lol. Yes. I'll update it in the next pass. Lots of little corrections to make. Thanks :)